docs/src/layouts/AppFooter.tsx

```diff
@@ -43,10 +43,10 @@ export default function AppFooter() {
         <div>
           <SvgMuiLogo width={32} />
           <Typography variant="body2" fontWeight="bold" sx={{ pt: 2 }}>
-            Join our newsletter!
+            Keep up with us
```

Comment:
Sounds like a race...

```suggestion
            Keep up to date
```